server/models/video/video-channel.ts

```diff
@@ -368,7 +410,16 @@ export class VideoChannelModel extends Model<VideoChannelModel> {
       ]
     }
 
+    const scopes: string | ScopeOptions | (string | ScopeOptions)[] = [ ScopeNames.WITH_ACTOR ]
```

Comment:
Why do we need `WITH_ACTOR` now?

Reply:
Because it corresponds to the default scope, which is not included as soon as we set a custom scope, so we have to include it manually.